### ISSUE TITLE
Highlight matching parentheses in math mode

### ISSUE THREAD

**Issue by fpvandoorn** [James-Yu/LaTeX-Workshop#292]
### Description

I really like highlighting of matching parentheses in vscode, i.e. if you navigate to one parenthesis, it and the matching parenthesis light up. In math mode this doesn't work with LaTeX-Workshop.

### Example

```latex
\documentclass{article}
\begin{document}
$ ( ) $ ( )
\[ ( ) \] ( )
\end{document}
```
The parentheses `( )` on the left are not highlighted when navigating to them but the parentheses `( )` on the right are.

**Comment by James-Yu** [comment]
I've also noticed this issue before, and iirc there were similar issues before. However it seems to be vscode related problem. I will leave this issue open and look for solutions.

**Comment by fpvandoorn** [comment]
This is fixed by #471. Thanks!
